src/main/com/example3/ArrayRNG.java

```diff
@@ -0,0 +1,37 @@
+package main.com.example3;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class ArrayRNG {
```

Comment:
Domluvme se, ze kazda classa, kde bude main metoda, tak bude koncit slovickem Applciation .. ArrayRNGApplication ... jinak to RNG znamena co?

Reply:
RNG = random number generator, ok přejmenuji

Reply:
ok, presto to prejmenuj, kdyby si kod precetla Edita, tak prvni otazka bude co je RNG ... tech dlouhych nazvu se v jave nemusis bat ... to je jen pro vzvoj, kdzy se to pak zbuildi, tak ten java compiler si to prejmenovava tz promenne na  a, b, c ... nebo nejak jinak

Reply:
na vsech mustech prosim

Reply:
Přejmenování pomocí Shift + F6

Reply:
jeste tam je metoda getRNG
a pak nejaka lokalni promenna rng

Reply:
Sorry, v novém commitu opraveno